src/services/QuoteService.ts

```diff
@@ -57,26 +50,16 @@ export class QuoteService {
     checkIn: Date,
     checkOut: Date
   ): Promise<BigNumber> => {
-    let losSpaceModifier: undefined | LOSRateModifier;
+    let losSpaceModifier: null | LOSRateModifier;
 
-    try {
-      losSpaceModifier = (await modifiers.getModifier(
-        'length_of_stay'
-      )) as LOSRateModifier;
-    } catch (e) {
-      if (e.status !== 404) {
-        throw e;
-      }
+    losSpaceModifier = await modifiers.getModifier<LOSRateModifier>(
+      'length_of_stay'
+    );
 
-      try {
-        losSpaceModifier = (await facilityModifiers.getModifier(
-          'length_of_stay'
-        )) as LOSRateModifier;
-      } catch (e) {
-        if (e.status !== 404) {
-          throw e;
-        }
-      }
+    if (!losSpaceModifier) {
+      losSpaceModifier = await facilityModifiers.getModifier<LOSRateModifier>(
```

Comment:
Suggest maybe call this variable `losModifier` as it may or may not be specific to the space.

Reply:
- [x] done